mifosng-android/src/main/java/com/mifos/mifosxdroid/formwidgets/FormSpinner.java

```diff
@@ -29,6 +29,7 @@ public FormSpinner(Context context, String name, List<String> columnValues, List
 
         label = new TextView(context);
         label.setText(getDisplayText());
+        label.setPadding(0, 16, 16, 0);
```

Comment:
Please remove hardcoded values from here, use dynamic values from the `dimens.xml` file.

Reply:
okay